packages/core/contracts/financial-templates/test/ExpiringMultiPartyMock.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "../../common/implementation/Testable.sol";
+import "../../common/implementation/FixedPoint.sol";
+import "../common/financial-product-libraries/FinancialProductLibrary.sol";
+
+
+contract ExpiringMultiPartyMock is Testable {
+    using FixedPoint for FixedPoint.Unsigned;
+
+    FinancialProductLibrary public financialProductLibrary;
+    uint256 public expirationTimestamp;
+
+    constructor(
+        address _financialProductLibraryAddress,
+        uint256 _expirationTimestamp,
+        address timerAddress
+    ) public Testable(timerAddress) {
+        expirationTimestamp = _expirationTimestamp;
+        financialProductLibrary = FinancialProductLibrary(_financialProductLibraryAddress);
+    }
+
+    function transformPrice(FixedPoint.Unsigned memory price, uint256 requestTime)
+        public
+        view
+        returns (FixedPoint.Unsigned memory)
+    {
+        if (address(financialProductLibrary) == address(0)) return price;
+        try financialProductLibrary.transformPrice(price, getCurrentTime()) returns (
```

Comment:
Should this be `requestTime` instead of `getCurrentTime()`?

Reply:
ye, it should be. have updated.